packages/ckeditor5-watchdog/package.json

```diff
@@ -20,6 +19,7 @@
     "@ckeditor/ckeditor5-paragraph": "^26.0.0",
     "@ckeditor/ckeditor5-theme-lark": "^26.0.0",
     "@ckeditor/ckeditor5-utils": "^26.0.0",
+    "ckeditor5": "^26.0.0",
```

Comment:
I'm not sure if we use it in tests at all. But I'm fine with it.

Reply:
DLL, webpack cfg.